x/stake/client/rest/rest.go

```diff
@@ -6,10 +6,17 @@ import (
 	"github.com/cosmos/cosmos-sdk/wire"
 
 	"github.com/gorilla/mux"
+	"github.com/gin-gonic/gin"
 )
 
 // RegisterRoutes registers staking-related REST handlers to a router
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *wire.Codec, kb keys.Keybase) {
 	registerQueryRoutes(cliCtx, r, cdc)
 	registerTxRoutes(cliCtx, r, cdc, kb)
 }
+
+// RegisterSwaggerRoutes - Central function to define stake related routes that get registered by the main application
+func RegisterSwaggerRoutes(routerGroup *gin.RouterGroup, ctx context.CLIContext, cdc *wire.Codec, kb keys.Keybase) {
+	registerSwaggerQueryRoutes(routerGroup, ctx, cdc)
+	registerSwaggerTxRoutes(routerGroup, ctx, cdc, kb)
```

Comment:
Please add the new staking endpoints as well #1880

Reply:
I have already added all endpoints in [#1880](https://github.com/cosmos/cosmos-sdk/pull/1880), please check registerSwaggerQueryRoutes
```
func registerSwaggerQueryRoutes(routerGroup *gin.RouterGroup, ctx context.CLIContext, cdc *wire.Codec) {
	routerGroup.GET("/stake/delegators/:delegatorAddr", delegatorHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/delegators/:delegatorAddr/txs", delegatorTxsHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/delegators/:delegatorAddr/validators", delegatorValidatorsHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/delegators/:delegatorAddr/validators/:validatorAddr", delegatorValidatorHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/delegators/:delegatorAddr/delegations/:validatorAddr", delegationHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/delegators/:delegatorAddr/unbonding_delegations/:validatorAddr", unbondingDelegationsHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/validators", validatorsHandlerCreation(cdc, ctx))
	routerGroup.GET("/stake/validators/:addr", validatorHandlerCreation(cdc, ctx))
}
```

Reply:
@HaoyangLiu you missed `GET /stake/pool` and ``GET /stake/parameters``